src/ShallowWrapper.js

```diff
@@ -321,7 +321,7 @@ export default class ShallowWrapper {
    * @returns {CheerioWrapper}
    */
   render() {
-    return cheerio.load(this.html()).root();
+    return this.type() === null ? cheerio() : cheerio.load(this.html()).root();
```

Comment:
did the previous implementation of this throw an exception in this case?  is `cheerio()` what we want to return?